README.md

```diff
@@ -244,48 +138,30 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project just give me a star ⭐.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+I would like to thank the follolowing: 
 
-I would like to thank...
+- [Microverse](https://www.microverse.org/)
+- [Cindy Chin - as this project is based on her design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```

Comment:
- [ ]  Instead of "Cindy Chin", it should be "Cindy Shin". Please correct that

Reply:
Fixed this.